.github/workflows/staging-build-macos.yml

```diff
@@ -0,0 +1,54 @@
+name: Build macOS using tar artifacts
+
+on:
+#  schedule:
+#    - cron: '0 12 * * *'
+  repository_dispatch:
+    types: [staging-build-macos]
+  push:
+    branches: [sreekarj_macos]
+
+jobs:
+  build-es-artifacts-x64:
+    name: Build macOS ES Artifacts X64
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Set Up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 14
+
+    - name: Required Packages
+      run: ./release-tools/scripts/required_packages.sh
+
+    - name: Build macOS
+      env:
+        install4j_license: ${{ secrets.INSTALL4J_LICENSE }}
+      run: ./elasticsearch/macos/opendistro-macos-build.sh macos x64
+       
+
+
+  build-kibana-artifacts-x64:
+    name: Build macOS KIBANA Artifacts x64
+    runs-on: macos-latest
```

Comment:
Better point to specific macos version instead of `macos-latest`.
Suggest `macos-10.15` to prevent the latest tag point to big sur (11) later on.
https://github.com/actions/virtual-environments/blob/main/README.md#available-environments

Reply:
I personally prefer going with the latest and greatest for the IT tests, as that will inform ahead of time if something is going to break. But I can change to macos-10.15 for the time being.

Reply:
Changed the build workflow to use macos-10.15. Will leave the IT to run on the latest, that will help us to know if there is a future version which causes issues.